src/PanelTraits/Breadcrumb.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Breadcrumb
+{
+    /*
+    |--------------------------------------------------------------------------
+    |                              BREADCRUMB
+    |--------------------------------------------------------------------------
+    */
+
+    protected $items;
+
+    /**
+     * Parse the word.
+     *
+     * @return string the parsed word.
+     **/
+    protected function parseWord($word)
+    {
+        $words = explode('_', $word);
+        $string = '';
+        foreach ($words as $key => $word) {
+            $string .= $key > 0 ? ' ' . ucfirst($word) : ucfirst($word);
+        }
+        return $string;
+    }
+
+
+    /**
+     * Generate the actual breadcrumb.
+     *
+     * @return string The actual breadcrumb to be displayed.
+     **/
+    public function buildBreadcrumb()
+    {
+        if (!$this->route) {
+          $this->setRoute(config('backpack.base.route_prefix'));
+        }
+
+        $this->items = new Collection(explode('.', $this->route));
+
+        $string = '';
+        foreach ($this->items as $key => $item) {
+            $string .= $this->getTag($key, $this->items, $item);
+        }
+        return $string;
+    }
+
+    /**
+     * Allow crumbs to be added at the beginning.
+     * @param string $item string of new item(s) to be added.
+     **/
+    public function addBreadcrumbStart($item)
+    {
+      $this->setRoute($item.'.'.$this->route);
+      $this->setBreadcrumb();
```

Comment:
Ok, that's definitely badly named. It needs to change...that sounds like it's `reinitializing` the breadcrumb maybe?